master/master.go

```diff
@@ -133,6 +198,114 @@ func SendTask(t *lib.Task, ip string) (*http.Response, error) {
 	return http.Post(url, lib.BodyType, bytes.NewBuffer(body))
 }
 
+func newEC2() *ec2.EC2 {
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String(lib.AWSRegion)},
+	)
+	if err != nil {
+		panic(err)
+	}
+	return ec2.New(sess)
+}
+
+func (m *Master) initAWS() {
+	m.svc = newEC2()
+
+	// create one slave on startup
+	s, err := createSlaves(m.svc, 1)
+	if err != nil {
+		fmt.Println("Failed to create slave")
+		panic(err)
+	}
+
+	ip := getPublicIP(m.svc, s[0])
```

Comment:
Should we do error handling here? What if we can't get the public IP in time? Do you have any other solution instead of using public IP as key?